src/config.h

```diff
@@ -199,6 +199,10 @@ typedef struct options {
 	bool animation_clamping;
 	/// TODO: window animation blacklist
```

Comment:
Now that a blacklist for window animations is done, please remove the TODO comment.